package exception.basic;

public class ExceptionLetitgo {
    public static void main(String[] args) {
        int[] arr = {1, 2, 3, 4, 5};

        System.out.println(arr[5]);

        System.out.println("나오나요!?");
    }
}
